traffic_ops/testing/api/v13/tc-fixtures.json

```diff
@@ -38,6 +38,11 @@
             "parentCachegroupName": "parentCachegroup",
             "secondaryParentCachegroupName": "secondaryCachegroup",
             "shortName": "cg1",
+            "localizationMethods": [
```

Comment:
Shouldn't these just be the .String() func call on your `LocalizationMethod`?

Reply:
Do you mean pre-processing/injecting this into the JSON using the String() methods before loading it for the tests?

Reply:
Just referring to the enum value instead of hard coding